RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/PFClusterProducerKernel.h

```diff
@@ -0,0 +1,93 @@
+#ifndef RecoParticleFlow_PFClusterProducerAlpaka_PFClusterProducerAlpakaKernel_h
+#define RecoParticleFlow_PFClusterProducerAlpaka_PFClusterProducerAlpakaKernel_h
+
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecHitHostCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFClusterDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitFractionDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/PFClusterParamsAlpakaESData.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/tmpDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/tmpEdgeDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  typedef struct float4 {
+    float x;
+    float y;
+    float z;
+    float w;
+  } float4;
+
+  typedef struct float3 {
+    float x;
+    float y;
+    float z;
+  } float3;
+
+  typedef struct int4 {
+    int x;
+    int y;
+    int z;
+    int w;
+  } int4;
+
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE float3 make_float3(float x, float y, float z) {
+    float3 tmp;
+    tmp.x = x;
+    tmp.y = y;
+    tmp.z = z;
+    return tmp;
+  }
```

Comment:
Are these functions necessary? The effect of
```cpp
make_float3(x, y, z);
```
can be achieved also with the aggregate initialization
```cpp
Float3{x,y,z};
```
(assuming the `float3` struct is renamed to `Float3` to follow our naming convention)